web/packages/teleport/src/Main/zIndexMap.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Teleport
+ * Copyright (C) 2024  Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+export const zIndexMap = {
```

Comment:
I'd say if something is supposed to be shared in a bunch of different places, we should push it down to a smaller "package"/module rather than up.

Right now I think it might count as a cyclic dependency, since items exported from `Main/index.ts` import stuff from `TopBarSideNav/TopBarSideNav.tsx`, which in turns imports `Main/index.ts` to get the z index map.

I know this map doesn't have a good spot and creating a separate file or a dir would seem a bit out of context. Overall, I feel like this z map belongs to the side nav and could be defined in it.

This means we'd end up with `TopBar/TopBarSideNav.tsx` importing `SideNavigation/Navigation.tsx` (which I mentioned here: https://github.com/gravitational/teleport/pull/46334#discussion_r1761127892), but it's still better as it's not a cyclic import and it lives closer to where it's used.